src/panels.cpp

```diff
@@ -1013,7 +1013,7 @@ static std::pair<translation, nc_color> weariness_description( size_t weariness
     return weary_descriptions[weariness];
 }
 
-static std::string activity_level_str( float level )
+const std::string activity_level::activity_level_str( float level )
```

Comment:
```suggestion
std::string activity_level::activity_level_str( float level )
```
Since `activity_level::activity_level_str` doesn't return a reference, it has no need to be `const`.